src/Bicep.Cli/Properties/launchSettings.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
@majastrz shouldn't this file be ignored?

Reply:
Good pt. I'll remove it in another PR.